master/main.go

```diff
@@ -55,7 +56,11 @@ func main() {
 
 	var err error
 
+	metriclogger.CreateMetricFolder()
+	go metriclogger.MonitorResourceUsage()
+
 	maxWorkers, err = strconv.Atoi(os.Getenv("MAXWORKERS"))
```

Comment:
Voor de apikey branch hebben we in de master ook een config struct die ingelezen wordt, dan kan daarin ook deze variabele toegevoegd worden.